tests/basictests/dsp-operator.sh

```diff
@@ -32,9 +32,32 @@ function create_and_verify_data_science_pipelines_resources() {
     os::cmd::try_until_text "oc get crd -n ${DSPAPROJECT} datasciencepipelinesapplications.datasciencepipelinesapplications.opendatahub.io" "datasciencepipelinesapplications.datasciencepipelinesapplications.opendatahub.io" $odhdefaulttimeout $odhdefaultinterval
     os::cmd::try_until_text "oc -n ${DSPAPROJECT} get pods -l app=ds-pipeline-sample -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}'" "True" $odhdefaulttimeout $odhdefaultinterval
     running_pods=$(oc get pods -n ${DSPAPROJECT} -l component=data-science-pipelines --field-selector='status.phase=Running' -o jsonpath='{$.items[*].metadata.name}' | wc -w)
+    echo "Sleeping for 5 minutes for the DSPO CR settle up "
+    sleep 5m
```

Comment:
not a huge fan of adding sleep, we should instead be checking for statuses/etc. but I also don't want to block this pr for ci tests problems, can we follow up this pr with the flowing: 

1. delegate this same change to the odh-manifest repo (we should keep these files the same in both repos) 
2. follow up with an issue (in this repo) to have a better check for dspo cr ready state.

Reply:
Opened a separate issue for that